From 6551103abd5143e51814ec1dce9b36bb9a46e09f Mon Sep 17 00:00:00 2001
From: chenrui <rui@meetup.com>
Date: Sat, 10 Aug 2024 14:42:15 +0000
Subject: [PATCH] fix gxml.pc.in

seeing version compare issue as below
```
Unknown version comparison operator '>=2.9.13' after package name 'libxml-2.0' in file '/opt/homebrew/lib/pkgconfig/gxml-0.20.pc'

  Unknown version comparison operator '>=0.20.6' after package name 'gee-0.8' in file '/opt/homebrew/lib/pkgconfig/gxml-0.20.pc'
```

relates to https://github.com/Homebrew/homebrew-core/pull/180780

Signed-off-by: Rui Chen <rui@chenrui.dev>
---
 gxml/gxml.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gxml/gxml.pc.in b/gxml/gxml.pc.in
index 38e5965..ebdf184 100644
--- a/gxml/gxml.pc.in
+++ b/gxml/gxml.pc.in
@@ -8,6 +8,6 @@ Name: libgxml
 Description: GObject API for XML manipulation and serialization library
 URL: http://live.gnome.org/GXml
 Version: @PROJECT_VERSION@
-Requires: glib-2.0 >= 2.72 gio-2.0 >= 2.72 gobject-2.0 >= 2.72 libxml-2.0 >=2.9.13 gee-0.8 >=0.20.6
+Requires: glib-2.0 >= 2.72 gio-2.0 >= 2.72 gobject-2.0 >= 2.72 libxml-2.0 >= 2.9.13 gee-0.8 >= 0.20.6
 Libs: -L${libdir} -lgxml-@API_VERSION@
 Cflags: -I${includedir}/gxml-@API_VERSION@
-- 
GitLab

